projects/plugins/jetpack/modules/sso.php

```diff
@@ -226,12 +226,28 @@ public function login_enqueue_scripts() {
 		}
 
 		if ( is_rtl() ) {
-			wp_enqueue_style( 'jetpack-sso-login', plugins_url( 'modules/sso/jetpack-sso-login-rtl.css', JETPACK__PLUGIN_FILE ), array( 'login', 'genericons' ), JETPACK__VERSION );
+			wp_enqueue_style(
+				'jetpack-sso-login',
+				plugins_url( 'modules/sso/jetpack-sso-login-rtl.css', JETPACK__PLUGIN_FILE ),
+				array( 'login', 'genericons' ),
+				JETPACK__VERSION
+			);
 		} else {
-			wp_enqueue_style( 'jetpack-sso-login', plugins_url( 'modules/sso/jetpack-sso-login.css', JETPACK__PLUGIN_FILE ), array( 'login', 'genericons' ), JETPACK__VERSION );
+			wp_enqueue_style(
+				'jetpack-sso-login',
+				plugins_url( 'modules/sso/jetpack-sso-login.css', JETPACK__PLUGIN_FILE ),
+				array( 'login', 'genericons' ),
+				JETPACK__VERSION
+			);
 		}
 
-		wp_enqueue_script( 'jetpack-sso-login', plugins_url( 'modules/sso/jetpack-sso-login.js', JETPACK__PLUGIN_FILE ), array( 'jquery' ), JETPACK__VERSION, false );
+		wp_enqueue_script(
+			'jetpack-sso-login',
+			plugins_url( 'modules/sso/jetpack-sso-login.js', JETPACK__PLUGIN_FILE ),
+			array(),
+			JETPACK__VERSION,
+			false
+		);
```

Comment:
This is mostly changes in indentation, for better readability. The only important change is the removal of the `jquery` dependency for the js file.